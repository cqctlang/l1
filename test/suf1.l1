foreach(@lambda(v) { printf("%a\n", v); },
	[ 1k, 1K, 1m, 1M, 1g, 1G, 1t, 1T,
	  1024k, 1024K, 1024m, 1024M, 1024g, 1024G, 1024t, 1024T ]);
