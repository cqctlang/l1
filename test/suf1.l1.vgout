1024
1024
1048576
1048576
1073741824
1073741824
1099511627776
1099511627776
1048576
1048576
1073741824
1073741824
1099511627776
1099511627776
1125899906842624
1125899906842624
