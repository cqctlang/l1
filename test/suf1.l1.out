entry: var params, 2 locals, 0 temps
L40: 1 params, 0 locals, 0 temps
1024
1024
1048576
1048576
1073741824
1073741824
1099511627776
1099511627776
1048576
1048576
1073741824
1073741824
1099511627776
1099511627776
1125899906842624
1125899906842624
